plugins/module_utils/_api_data.py

```diff
@@ -1519,6 +1519,37 @@ def join_path(path):
             },
         ),
     ),
+    ('interface', 'l2tp-client',): APIData(
+        unversioned=VersionedAPIData(
+            single_value=False,
+            fully_understood=True,
+            fields={
+                'add-default-route': KeyInfo(default=False),
+                'allow': KeyInfo(default='pap,chap,mschap1,mschap2'),
+                'allow-fast-path': KeyInfo(default=False),
+                'comment': KeyInfo(can_disable=True, remove_value=''),
+                'connect-to': KeyInfo(),
+                'default-route-distance': KeyInfo(default=False),
+                'dial-on-demand': KeyInfo(default=False),
+                'disabled': KeyInfo(default=True),
+                'ipsec-secret': KeyInfo(default=''),
+                'keepalive-timeout': KeyInfo(default=60),
+                'l2tp-proto-version': KeyInfo(default='l2tpv2'),
+                'l2tpv3-cookie-length': KeyInfo(default=0),
+                'l2tpv3-digest-hash': KeyInfo(default='md5'),
+                'max-mru': KeyInfo(default=1450),
+                'max-mtu': KeyInfo(default=1450),
+                'mrru': KeyInfo(default='disabled'),
+                'name': KeyInfo(),
```

Comment:
Does `name` or any other field (or combination of fields) happen to be a primary key? (I.e. for every value there can be at most one entry in this path where this field has that value.)

Reply:
Added primary_keys=('name', ) and changed which fields required=True